llvm/docs/LangRef.rst

```diff
@@ -11756,10 +11756,10 @@ If the source is :ref:`poison <poisonvalues>`, the result is
 :ref:`poison <poisonvalues>`.
 
 If the source is not :ref:`poison <poisonvalues>`, and both source and
-destination are :ref:`integral pointers <nointptrtype>`, and the
-result pointer is dereferenceable, the cast is assumed to be
-reversible (i.e. casting the result back to the original address space
-should yield the original bit pattern).
```

Comment:
None of the text here should be removed. You dropped the important phrasing about reversibility. This is *not* changing this to assume it's truncated / extended bitcast. You're only assuming the alignment low bits are preserved through integral addrspacecasts